TeamCode/src/main/java/org/firstinspires/ftc/teamcode/modules/robot/Claw.java

```diff
@@ -1,41 +1,59 @@
 package org.firstinspires.ftc.teamcode.modules.robot;
 
 import com.qualcomm.robotcore.hardware.Servo;
+import com.qualcomm.robotcore.util.ElapsedTime;
 
 import org.firstinspires.ftc.robotcore.external.Telemetry;
 import org.firstinspires.ftc.teamcode.opmode.config.ClawConfig;
 
 public class Claw extends BaseComponent {
-    static final float TOP_CLOSE = 1;
-    static final float TOP_OPEN = 0.5f;
+    static final float TOP_CLOSE = 0;
```

Comment:
It would be good to see code comments referring to what open and close mean for each so if I am reading the code I can go look at the bot and understand.